### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Jaime-Burgos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,19 +82,19 @@ topics:
...
+*The dynamic types standard does not allow certain characters in its names*. For this reason, if a type defined in the topics section of the configuration file has in its name a `/`, the dds system handle will map that character into two underscores. That's why the type inside the dynamic types map is std_msgs__String, while the type inside the topics section is std_msgs/String. This is *something important to notice when connecting to ROS2*, because in ROS2 most of the types have a `/` in their names. Also, notice that *in the DDS system, the message will be published with a type name with two underscores instead of slashes*.
```
Comment (lemunozm) [inline README.md]: Please, divide each sentence to different lines to improve the git diffs. (As a rule, each dot division must create a new line. If any sentence is too long, also must be divided even if it has not dot).
* This comment applies to any other paragraph.

File: README.md [hunk 2]
```diff
@@ -88,19 +82,19 @@ topics:
...
 
 On the other hand, `participant` map tells the dds system handle where it can find the configuration file for the DDS profle, and what profile must be used from the many that can be defined in that XML. This profile is used to set the DDS quality of services' parameters. A guide on how this XML files are configured can be found in [Fast-RTPS' documentation](https://fast-rtps.docs.eprosima.com/en/v1.7.2/xmlprofiles.html). An example of an XML configuration file can be found [in this repository](dds/sample/tcp/config.xml). Notice that this example file has two participant profiles defined in it, one to be used in the client side and other for the server side, so the YAML file used to configure SOSS in the server computer must change the profile_name in the example above from "soss_profile_client" to "soss_profile_server".
 
-The `participant` map is optional, and if it is not given, the dds system handle will create a default UDP profile.
+The `participant` map is optional, and if it is not given, the dds system handle will create a default UDP profile that can communicate in a LAN.
```
Comment (lemunozm) [inline README.md]: Exists some case where you can communicate by udp out of the LAN?

File: README.md [hunk 3]
```diff
@@ -52,18 +54,10 @@ To install this package into a workspace already containing SOSS, just clone thi
...
-
-If we take as an example the communication between ROS2 and FIWARE, the communication scheme will look like this:
-
-![](dds/doc/images/ROS2_TCP_tunnel.png)
-
 ### Configuration
 
 SOSS must be configured with a YAML file, which tells the program everything it needs to know in order to establish the connection between two or more systems that the user wants. 
-For example, if a simple string message wants to be exchanged between two ROS2 systems through a TCP tunnel, the configuration files for the two different SOSS instances that that tunnel must have should look as follows.
+For example, if a simple string message wants to be exchanged between two ROS2 systems through a [TCP tunnel](#tcp-tunnel), the configuration files for the two different SOSS instances that that tunnel must have should look as follows.
```
Comment (lemunozm) [inline README.md]: - double `that` word.
- as follows following by `:`

File: README.md [hunk 4]
```diff
@@ -4,8 +4,10 @@ System handle to connect [*SOSS*][soss] to *eProsima*'s open-source implementati
 
 ## Installation
 To install this package into a workspace already containing SOSS, just clone this repository into the sources directory and build it:
+
     ```
     git clone git@github.com:eProsima/SOSS-DDS.git
+
     (in the root of the workspace) colcon build --packages-up-to soss-dds
     ```
 ## Use case - Connecting with ROS2
```
Comment (lemunozm) [inline README.md]: This is a personal opinion: `Use case` is too formal for me,  it appears to be related with a detailed document of purposes of the application. Instead, I think that `Example of connection with ros2` is more attractive for a person that just arrives at the repo and start to read. This kind of person will try to find the `example` word.

File: README.md (hunk 1 above)
Comment (Jaime-Burgos) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Jaime-Burgos) [reply README.md]: erased

File: README.md (hunk 3 above)
Comment (Jaime-Burgos) [reply README.md]: corrected

File: README.md (hunk 4 above)
Comment (Jaime-Burgos) [reply README.md]: Changed

**Review by lemunozm — APPROVED** [review]

**Pull request merged**